overlord/swfeats/registry_test.go

```diff
@@ -121,7 +121,7 @@ func (s *swfeatsSuite) TestDuplicateAdd(c *C) {
 }
 
 func (s *swfeatsSuite) TestCheckChangeRegistrations(c *C) {
-	goFiles, err := getGoFiles("../../", "/snapstatetest/")
+	goFiles, err := getGoFiles("../../", "/snapstatetest/", "../../.git/", "../../vendor/", "../../tests/")
```

Comment:
I'd add `data`, `packaging`, `release-tools`, `po`.
```suggestion
	exclusions := []string{
		"/snapstatetest/",
		 "../../.git/",
		 "../../vendor/", // vendored dependencies
		 "../../tests/",
		 "../../packaging", "../../debian", // distro packaging
		 "../../data", // static data
		 "../../release-tools", // release scripts
		 "../../po", // translations
	}
	goFiles, err := getGoFiles("../../", exclusions...)
```